scope

```diff
@@ -9,7 +9,14 @@ usage() {
     exit 1
 }
 
-SCOPE_IMAGE=weaveworks/scope
+SCRIPT_VERSION="(unreleased version)"
+if [ "$SCRIPT_VERSION" = "(unreleased version)" ] ; then
+    IMAGE_VERSION=latest
+else
+    IMAGE_VERSION=$SCRIPT_VERSION
+fi
```

Comment:
Line 12 gets rewritten by the release script.  This is just a copy of the weave script.